person_class.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age > 18
```

Comment:
- [ ] Kindly check your of_Age method, is should check if the person is 18 years and above currently it's only checking if they are above 18 years.

Reply:
done